core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -780,6 +781,22 @@ public List<LogRecord> getLogRecords() throws IOException, InterruptedException
             return channel.call(new SlaveLogFetcher());
     }
 
+    /**
+     * Inline editing of description
+     */
+    @RequirePOST
+    public synchronized void doSubmitDescription(StaplerResponse rsp, @QueryParameter String description) throws IOException {
+        checkPermission(CONFIGURE);
+
+        final Slave node = this.getNode();
+        if (node != null) {
+            node.setNodeDescription(description);
+        } else { // Node has been disabled/removed during other session tries to change the description.
+            throw new IOException("Node " + nodeName + " has been deleted during other session tries to change the description");
```

Comment:
It is unclear to me what these messages mean.

Reply:
It might happens, that you edit the description, but the node is deleted between (some other user or you in other session, or some groovy script or ... ). Any better English translations are welcome.

Reply:
I will propose improvements 👍

Reply:
Make it easy. I think it is more sure now.